<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QMainWindow" name="TextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir_archivo"/>
    <addaction name="actionCerrar_archivo"/>
    <addaction name="actionGuardar_archivo"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionBuscar"/>
    <addaction name="actionReemplazar"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
  </widget>
  <action name="actionAbrir_archivo">
   <property name="text">
    <string>Abrir archivo...</string>
   </property>
  </action>
  <action name="actionCerrar_archivo">
   <property name="text">
    <string>Cerrar archivo</string>
   </property>
  </action>
  <action name="actionGuardar_archivo">
   <property name="text">
    <string>Guardar archivo</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionReemplazar">
   <property name="text">
    <string>Reemplazar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
